<launch> 
    <node name="action_handler" pkg="dmp_vfc" type="action_handler.py" respawn="true" output="screen"/>
</launch>
